### ISSUE TITLE
Information on frame decoded fail

### ISSUE THREAD

**Issue by scaret** [mbebenita/Broadway#168]
I suppose that every successfully decoded I frame or P frame will trigger the event `onFrameDecoded` once. Is there corresponding event `onFrameDecodedFail` event?

**Comment by hosnar** [comment]
Hi. Did you manage to figure this out?

**Comment by scaret** [comment]
@hosnar No. I kept maximum 2 frames in decoder and set a timeout so as to reset the decoder if it has no response.
